queries/elixir/aerial.scm

```diff
@@ -57,3 +57,18 @@
     target: (identifier) @identifier (#eq? @identifier "defstruct")) @type
   (#set! "kind" "Function")
   ) @start
+
+; describe("Unit test")
+(call
+  target: (identifier) @identifier (#any-of? @identifier "describe" "test")
+  (arguments [(string (quoted_content) @name)])
+  (#set! "kind" "Function")
+  ) @type
+
+; describe("Unit test")
```

Comment:
Copypasta?

Reply:
I saw this elsewhere in aerial and thought it might have special meaning. Happy to remove it or format it differently